fix : Changes in job applications

Nidhashirinniyas opened the pull request:
## Issue description
- Some Changes in Job Application Form Design
- Address field and previous employee details

## Solution description
- Design Changed as per required 
- Added a function that if country is India then Adhaar no is required
- 
## Output screenshots (optional)
<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-22-30" src="https://github.com/user-attachments/assets/55abe350-e4e3-4d8d-b369-48383f061a15" />
<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-22-56" src="https://github.com/user-attachments/assets/f084225e-2d38-4240-873f-0e8ef5b53ee9" />
<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-23-29" src="https://github.com/user-attachments/assets/1b639baf-3005-4100-9525-3d2cca92cdf8" />
<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-23-57" src="https://github.com/user-attachments/assets/2e366fad-3765-4b11-a125-c2f11f932e02" />
<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-24-08" src="https://github.com/user-attachments/assets/7f736601-7c94-4c30-85ba-4752b0504d7b" />

<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-23-45" src="https://github.com/user-attachments/assets/173dc25b-9426-4a88-bd3a-7045f3a449ce" />

<img width="1920" height="1080" alt="Screenshot from 2025-07-15 21-23-18" src="https://github.com/user-attachments/assets/d777876e-506b-4849-bce5-9b5b5e340473" />

Nidhashirinniyas committed:
fix : Changes in job applications

Nidhashirinniyas committed:
fix : Added some more changed in job application form

MhmdSinanKT reviewed (changes requested):
this only has the html changes; please include the respective changes required to store the data collected from this page

SherinKR commented on beams/www/job_application_upload/upload_doc.html:
```diff
@@ -600,6 +740,41 @@ <h5><b>Declaration and Authorization </h3></b>
+
+            // Show/hide Aadhaar field based on country
+            countrySelect.addEventListener('change', function () {
+                if (this.value === "India") {
```
What if india and India

SherinKR reviewed (dismissed):
Check entire field names used for labels, include the js changes to capture the values

Nidhashirinniyas committed:
fix : Validating and Fetching data

MhmdSinanKT reviewed (changes requested):
Coding Standards

Nidhashirinniyas committed:
chore:Coding standard

MhmdSinanKT commented on beams/setup.py:
Indentation

MhmdSinanKT commented on beams/www/job_application_upload/upload_doc.py:
```diff
@@ -28,130 +29,196 @@ def authorize_applicant_id(encrypted_applicant_id):
 
 @frappe.whitelist(allow_guest=True)
-def update_register_form(docname, data):
-	'''
```
don't remove the documentation

Nidhashirinniyas committed:
fix:Documentation added and redirect corrected

ZakkiyathLulu committed:
refactor: Clean up imports and standardize variable assignments

- Rearranged import statements for better organization.
- Removed redundant imports and ensured only necessary modules are included.
- Standardized the assignment of document fields to improve readability.
- Fixed minor typos in variable names for consistency.
